pkg/lambdaworks/lambdaworks_test.go

```diff
@@ -167,3 +167,27 @@ func TestFeltDiv4Error(t *testing.T) {
 		t.Errorf("TestFeltDiv4Error failed. Expected: %v, Got: %v", expected, result)
 	}
 }
+
+func TestFeltIsZero(t *testing.T) {
+	f_zero := lambdaworks.FeltZero()
+
+	expected := true
+
+	result := f_zero.IsZero()
+
+	if result != expected {
+		t.Errorf("TestFeltIs0 failed. Expected: %v, Got %v", expected, result)
+	}
+}
+
+func TestFeltisNotZero(t *testing.T) {
+	f_one := lambdaworks.FeltOne()
+
+	expected := false
+
+	result := f_one.IsZero()
+
+	if result != expected {
```

Comment:
This can be simplified:

```suggestion
	is_zero := f_one.IsZero()

	if is_zero {
```